Aikido.Zen.Core/Patches/IOPatcher.cs

```diff
@@ -62,5 +70,50 @@ public static bool OnFileOperation(string[] paths, MethodBase originalMethod, Co
             }
             return true;
         }
+
+        private static bool ShouldExcludeAssembly(string assemblyFullName)
```

Comment:
Isnt this more something we could add on a global level, instead of in the individual patch?